tunnel/intra/split/example/main.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"crypto/tls"
+	"log"
+	"net"
+	"os"
+
+	"github.com/Jigsaw-Code/outline-go-tun2socks/tunnel/intra/split"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		log.Printf("Usage: %s destination [SNI]", os.Args[0])
+		log.Printf("This tool attempts TLS connection to the destination (port 443), with and without splitting.")
+		log.Printf("If the SNI is specified, it overrides the destination, which can be an IP address.")
+		return
+	}
+	destination := os.Args[1]
+	addr, err := net.ResolveTCPAddr("tcp", net.JoinHostPort(destination, "443"))
+	if err != nil {
+		log.Println("Couldn't resolve destination")
```

Comment:
Use `log.Fatal` to avoid the return statement and exit with non-zero status code.

Reply:
Done